doc/sphinx/source/recipes/recipe_cvdp.rst

```diff
@@ -0,0 +1,30 @@
+CVDP Package
+============
+
+About
+-----
+
+The NCAR Climate Variability Package (CVDP) is a stand alone ncl based application for the analysis of climate variability in models and observations, see [1].
+
+Requirements
+------------
+
++ cvdp (https://github.com/NCAR/CVDP-ncl)
++ nco (optional for creating netcdf files)
+
+Configuration
+-------------
+
+To use it within the ESMValTool set the environment variable *CVDP_ROOT* to the top level directory of the CVDP package.
```

Comment:
It would be nice if this could also be done by the wrapper script. If we include a copy of the package with esmvaltool, e.g. in the same directory as the wrapper script, this should be easy.

Reply:
I am not sure if we can include this, see comment below (License etc.)

I see myself making a conda package for this, or to let the setup.py run some kind of wget/tar/setenv-operation. Is that possible?

Reply:
The package now has an Apache 2 license, so we can just include a copy with esmvaltool.

Reply:
I added the cvdp package on the same level as the `cmor` folder. I hope that's the right place. If we come with many more external packages, we might want to find a better place.

Reply:
Can you move it to the esmvaltool/diag_scripts/cvdp directory? That way it is located closer to where it is needed.

Reply:
I would also keep the license file with the package.

Reply:
I moved the external package to `esmvaltool/diag_scripts/cvdp` as suggested. The original license file is also there: `esmvaltool/diag_scripts/cvdp/cvdp/LICENSE`